test/unit/media/media.test.ts

```diff
@@ -1482,6 +1482,18 @@ describe('SdkMedia', () => {
       expect(sdk.sessionManager.validateOutgoingMediaTracks).toHaveBeenCalled();
       expect(sdk.logger.debug).toHaveBeenCalled();
     });
+
+    it('should not call sessionManager if it has not been initialized yet', async () => {
+      const enumerateDevicesSpy = jest.spyOn(sdkMedia, 'enumerateDevices').mockResolvedValue([]);
+      const miniSdk = { logger: { debug: jest.fn() } };
+      sdkMedia['sdk'] = miniSdk as any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9596369)
```suggestion
      sdkMedia['sdk'] = miniSdk as unknown;
```